app/main/views/frameworks.py

```diff
@@ -44,10 +43,25 @@ def framework_dashboard(framework_slug):
     framework = get_framework(data_api_client, framework_slug, open_only=False)
 
     if request.method == 'POST':
+        register_interest_in_framework(data_api_client, framework_slug)
+        supplier_users = data_api_client.find_users(supplier_id=current_user.supplier_id)
+
         try:
-            register_interest_in_framework(data_api_client, framework_slug)
-        except APIError as e:
-            abort(e.status_code)
+            email_body = render_template('emails/dos_application_started.html')
```

Comment:
Email template is specific to dos, we'd have to make it part of the frameworks content to make this generic